src/localization/locales/en-US.json

```diff
@@ -244,15 +244,28 @@
         "form_boundary_geojson_title": "Upload your landscape’s boundary",
         "form_location_placeholder": "Landscape location",
         "form_location_select": "Select location",
-        "form_name_placeholder": "Your landscape’s official name"
+        "form_name_placeholder": "Your landscape’s official name",
+        "boundaries_file_no_accepted": "No accepted files",
+        "shared_data_upload_title": "Share files with “{{name}}“"
     },
     "shared_files": {
         "title": "Shared files",
         "description": "Only the members of your group can download the files. Read more about sharing files.",
         "upload_button": "Upload Files",
         "delete_confirm_title": "Delete “{{name}}”?",
         "delete_confirm_message": "Are you sure you wish to delete the file “{{name}}”?",
-        "delete_confirm_button": "Delete File"
+        "delete_confirm_button": "Delete File",
+        "upload_file_success": "[TODO] File upload succesfuly",
```

Comment:
question: is this the inline success message (above the file name)?

Reply:
@paulschreiber yes, this will be the succes message on top of each file